pkg/kubelet/eviction/helpers.go

```diff
@@ -93,8 +95,17 @@ func validSignal(signal evictionapi.Signal) bool {
 }
 
 // ParseThresholdConfig parses the flags for thresholds.
-func ParseThresholdConfig(evictionHard, evictionSoft, evictionSoftGracePeriod, evictionMinimumReclaim string) ([]evictionapi.Threshold, error) {
+func ParseThresholdConfig(includeAllocatableThreshold bool, evictionHard, evictionSoft, evictionSoftGracePeriod, evictionMinimumReclaim string) ([]evictionapi.Threshold, error) {
 	results := []evictionapi.Threshold{}
+	if includeAllocatableThreshold {
+		results = append(results, evictionapi.Threshold{
```

Comment:
I am not sure I understand either point.
The threshold here on or off based on includeAllocatableThreshold.
Why would I tie allocatable limits to available memory?